### ISSUE TITLE
Logged-in Users are seeing incorrect call counts

### ISSUE THREAD

**Issue by nickoneill** [5calls/ios#227]
A few complaints from iOS folks recently that they've signed in and their call counts have dropped from their previous totals.

At first I thought this was the backend code for uploading stats for users, but it appears that it works fine. For example, here's the stats I see on the backend for a user:

```
{
  "stats": {
    "contact": 316,
    "voicemail": 506,
    "unavailable": 27
  },
  "weeklyStreak": 35
}
```

They report that the impact screen shows 35 week streak (correctly), but only 3 calls made. Perhaps the stats are being displayed based on local data and not remote data?

**Comment by subdigital** [comment]
Indeed that is the case. It's just looking at the saved pantry store. Looking at where this data is fetched / stored locally.

**Comment by nickoneill** [comment]
There's a call to `https://api.5calls.org/v1/users/stats` which is where the remote stats are coming from, so we should be getting the right data there.

I know we opted to keep some features relying on local data... if this isn't working correctly it might be worth using the remote data entirely.

**Comment by subdigital** [comment]
On 2nd look, I take that back. It is merging these 2 types of data, but the server stats are not being fetched on this screen. I think I can get a quick PR up to address this.

**Comment by subdigital** [comment]
Ah, it was fetching, but in a roundabout way (posting a profile changed notification manually).

**Comment by subdigital** [comment]
Made some changes while looking into this #228, it's working for me so I'm not sure what else to do for now. Everything looks okay.

**Comment by nickoneill** [comment]
Merged #228 in for this. Thanks @subdigital
